doc/cheatsheet.mld

```diff
@@ -66,12 +66,20 @@ Quick reference for the odoc language!
 
       See {{!Odoc_odoc.Compile.compile}this function}.
 
+      See {{!/fmt/Fmt.pf}this function from another library}.
+
+      See {{!/cmdliner/tutorial}this page from another package}.
```

Comment:
This reference doesn't work when building the doc with the driver.

Reply:
Hum that is strange, it should...

Reply:
I redid the test from scratch and the reference works. Are you sure you tested it right?